primitives/src/currency.rs

```diff
@@ -200,11 +200,11 @@ create_currency_id! {
 		// 149: Reserved for renBTC
 		// 150: Reserved for CASH
 		// 168 - 255: Kusama parachain tokens
-		BNC("Bifrost Native Token", 12) = 168,
-		VSKSM("Bifrost Voucher Slot KSM", 12) = 169,
-		PHA("Phala Native Token", 12) = 170,
-		KINT("Kintsugi Native Token", 12) = 171,
-		KBTC("Kintsugi Wrapped BTC", 8) = 172,
+		// BNC("Bifrost Native Token", 12) = 168,
```

Comment:
This will break compatibility for both SDK and existing runtime storages (e.g. BNC swap pair)
It should be possible to do some migration but it isn't that much beneficial to migrate.